dcos/dcos-ui
# DCOS-48334: environment form section

Add environment form section to jobs form.

## Testing

Using an OSS cluster:
- Open jobs form, go to environment tab
- Try adding/deleting environment variables. Ensure they are added/deleted from JSON
- Try testing validation

## Trade-offs



## Dependencies

DCOS-48334 will not be considered complete without the EE secrets.

## Screenshots

<img width="1401" alt="Screen Shot 2019-04-18 at 3 32 01 PM" src="https://user-images.githubusercontent.com/19582796/56395367-29ed3400-61ef-11e9-8c02-647290687de4.png">
